.yamato/project.metafile

```diff
@@ -50,9 +50,9 @@ projects:
       - 2021.2
       - 2020.3
       - 2022.1
-#  - name: testproject-tools-integration
-#    path: testproject-tools-integration
-#    validate: false
-#    test_editors:
-#      - 2021.2
-#      - trunk
+  #- name: testproject-tools-integration
+  #  path: testproject-tools-integration
+  #  validate: true
```

Comment:
Not sure I understand this change, if the job is still commented out, was the change from false to true necessary?

Reply:
No that will get reverted out... good catch.